make_profiler/parser.py

```diff
@@ -127,3 +129,47 @@ def recurse_indirect_influences(original_target, recurse_target):
             recurse_indirect_influences(original_target, t)
 
     return dependencies, influences, order_only, indirect_influences
+
+
+def check_include_instruction(filename):
```

Comment:
I would expect this to be a part of parse() function.

Reply:
moved inside parse() function